packages/nextjs/src/components/_common/chatbot/ChatbotLinkPanel.module.scss

```diff
@@ -1,22 +1,21 @@
-$chatPointerSize: 1.5rem;
-
 .chat {
-    $bgColor: var(--a-white);
-    $borderRadius: 24px;
-
     position: relative;
-    background-color: $bgColor;
-    border-radius: $borderRadius $borderRadius $borderRadius 0;
+    background-color: var(--a-surface-default);
+    border-radius: var(--a-spacing-6) var(--a-spacing-6) var(--a-spacing-6) 0;
```

Comment:
Det finnes ikke noe Aksel-token for 24px radius, så her brukte jeg `Spacing-6`. Vet ikke om det er lurt eller dumt 🤔